ompi/mca/coll/libnbc/nbc_iallreduce.c

```diff
@@ -9,7 +9,7 @@
  *                         reserved.
  * Copyright (c) 2014-2018 Research Organization for Information Science
  *                         and Technology (RIST).  All rights reserved.
- * Copyright (c) 2017      IBM Corporation.  All rights reserved.
+ * Copyright (c) 2022      IBM Corporation.  All rights reserved.
```

Comment:
This should be in the form of ` * Copyright (c) 2017-2022       IBM Corporation. All rights reserved.`